ms_game_chat/backend/backend/settings.py

```diff
@@ -37,7 +35,7 @@
 JWT_SECRET = os.environ.get('JWT_SECRET')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
@mariekart42 can you fetch the variable from the env here?

Reply:
will do in next PR